pose_1:
  servo1: 90
  servo2: 45
  servo3: 135
  servo4: 180
pose_2:
  servo1: 45
  servo2: 90
  servo3: 45
  servo4: 90
pose_3:
  servo1: 0
  servo2: 0
  servo3: 0
  servo4: 0
pose_4:
  servo1: 170
  servo2: 120
  servo3: 23
  servo4: 32